GameAnalytics/Source/GameAnalyticsEditor/GameAnalyticsEditor.Build.cs

```diff
@@ -29,7 +29,7 @@ public GameAnalyticsEditor(TargetInfo Target)
                     "SharedSettingsWidgets",
                     "SourceControl",
                     "UnrealEd",
-                    "Http",
+                    "HTTP",
```

Comment:
UBT complains that "Http" is not the correct case for this module (at least on macOS)